### PR TITLE
Add meal sharing application database queries.

### PR DESCRIPTION
**Pull request by GreeshmaRahool**
Added databases/week3 homework.

### CODE REVIEW

File: databases/week3/meal_sharing.sql [hunk 1]
```diff
@@ -0,0 +1,290 @@
+CREATE DATABASE meals;
+SET
+  NAMES utf8mb4;
```
Comment (sboutzen) [inline databases/week3/meal_sharing.sql:3]: No need for a new line here

File: databases/week3/meal_sharing.sql [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT NULL DEFAULT NULL,
+  `meal_id` INT(10) UNSIGNED NOT NULL,
+  `stars` INT(10) NOT NULL,
+  `created_date` DATETIME NOT NULL DEFAULT NOW(),
+  CONSTRAINT `fk_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+INSERT INTO
+  meal (
```
Comment (sboutzen) [inline databases/week3/meal_sharing.sql:39]: All of these newlines decrease readability imho. Maybe have each entry be on a single line

File: databases/week3/meal_sharing.sql [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+FROM meal
+WHERE title LIKE '%salmon%';
+--Get meals that has been created between two dates--
+SELECT title , created_date
+FROM meal
+WHERE created_date BETWEEN '2021-08-10 10:00:00' AND '2021-08-12 10:00:00';
+--Get only specific number of meals fx return only 5 meals--
+SELECT title,description
+FROM meal
+LIMIT 3;
```
Comment (sboutzen) [inline databases/week3/meal_sharing.sql:273]: A newline before each comment would make it easier on the eyes

**Review by sboutzen — COMMENTED** [review]
Very well done, small comments

File: databases/week3/meal_sharing.sql [hunk 4]
```diff
@@ -0,0 +1,290 @@
+CREATE DATABASE meals;
+SET
+  NAMES utf8mb4;
+USE meals;
+CREATE TABLE `meal`
```
Comment (sboutzen) [inline databases/week3/meal_sharing.sql:5]: Maybe put the table creations in a separate file to make it easier to get an overview of db structure vs the queries you are making

File: databases/week3/meal_sharing.sql (hunk 2 above)
Comment (GreeshmaRahool) [reply databases/week3/meal_sharing.sql:39]: Thank you so much for the great comments. I have updated all the feedback.

**Pull request merged**